src/client/src/components/LoginCard/index.tsx

```diff
@@ -29,32 +30,24 @@ const LoginCard = ({
   option
 }: IProps) => {
   return (
-    <div className={styles.loginContainer}>
-      <div className={styles.cardTitle}>
-        <img className={styles.icon} src={svgUrl} alt="" />
-        {cardTitle}
-      </div>
-      <div className={styles.cardBody}>
-        <CardBody body={cardBody} />
-      </div>
-      <div className={styles.selectionContainer}>
-        <Link
-          className={styles.details}
-          onClick={() => handleDetailsClick(option)}
-          to={ROUTES.PAGE_DETAILS}
-        >
-          <FormattedMessage id="loginCard.details" defaultMessage="Details" />
-        </Link>
-        <div className={styles.buttonContainer}>
-          <a
-            className={styles.createAccountButton}
-            href="https://azure.microsoft.com/free/"
+    <div className={styles.loginPrompt}>
+      <div className={styles.loginContainer}>
+        <div className={styles.cardTitle}>
+          <img className={styles.icon} src={svgUrl} alt="" />
```

Comment:
Out of curiosity, why are we using `svg` url instead of directly importing the `.svg` file like in Nipun's `-`/`+` PRs?

Reply:
The way we do svgs in the client right now is really consistent. Sometimes we use `src` and sometimes import it as a component. So probably a good idea to clean this up sometime

Reply:
From what I can remember I think we did svgUrl for things that are/can be dynamic, and raw import for static assets.

Reply:
As of now, it will always be the same icon because we only use it once. But it would be helpful if we need to reuse the same component but change the svg